psci.el

```diff
@@ -128,8 +128,12 @@ Relies on .psci file for determining the project's root folder."
         ;; create the comint process if there is no buffer.
         (unless buffer
           (setq default-directory (psci/--project-root!))
-          (apply 'make-comint-in-buffer psci/buffer-name buffer
-                 psci-program nil psci/arguments)
+          (let ((full-arg-list (if (file-exists-p "psc-package.json")
+                                   (let ((psc-package-sources (split-string (shell-command-to-string "psc-package sources"))))
```

Comment:
Might make sense to extract this into its own function. Something like `psci/get-psc-package-sources`.